app/api/custom/orders.py

```diff
@@ -93,3 +101,66 @@ def calculate_amount():
             return UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
 
     return jsonify(calculate_order_amount(tickets, discount_code))
+
+
+@order_blueprint.route('/create-order', methods=['POST'])
+@jwt_required
+def create_order():
+    data = request.get_json()
+    tickets = data['tickets']
+    attendee = data['attendee']
+    for attribute in attendee:
+        attendee[attribute.replace('-', '_')] = attendee.pop(attribute)
+    schema = AttendeeSchema()
+    json_api_attendee = {"data": {"attributes": data['attendee'], "type": "attendee"}}
+    result = schema.load(json_api_attendee)
+    if result.errors:
+        return jsonify(result.errors)
+    for ticket in tickets:
+        ticket_info = db.session.query(Ticket).filter_by(id=int(ticket['id']), deleted_at=None).first()
```

Comment:
Do this in a single query `filter(ids in ticket_ids, event_id=event_id, deleted_at=None)`